features/networking/multus-ipv6.feature

```diff
@@ -0,0 +1,55 @@
+Feature: Multus-CNI ipv6 related scenarios  
+  
+  # @author weliang@redhat.com
+  # @case_id OCP-28968
+  @admin
+  Scenario: IPv6 testing for OCP-21151: Create pods with multus-cni - macvlan bridge mode
+    # Make sure that the multus is enabled
+    Given the master version >= "4.1"
+    And the multus is enabled on the cluster
+    Given the default interface on nodes is stored in the :default_interface clipboard
+    And evaluation of `node.name` is stored in the :target_node clipboard
+    # Create the net-attach-def via cluster admin
+    Given I have a project
+    When I run oc create as admin over "https://raw.githubusercontent.com/weliang1/Openshift_Networking/master/Features/multus/IPv6/macvlan-bridge-v6.yaml" replacing paths:
+      | ["metadata"]["namespace"] | <%= project.name %>                                                                                                                                                                                                                             |    
+      | ["spec"]["config"]| '{ "cniVersion": "0.3.0", "type": "macvlan", "master": "<%= cb.default_interface %>","mode": "bridge", "ipam": { "type": "host-local", "subnet": "fd00::192:168:22:0/112", "rangeStart": "fd00::192:168:22:100", "rangeEnd": "fd00::192:168:22:200"} }' |
+    Then the step should succeed
+
+    # Create the first pod which consumes the macvlan custom resource
+    When I run oc create over "https://raw.githubusercontent.com/weliang1/Openshift_Networking/master/Features/multus/IPv6/1interface-macvlan-bridge-v6.yaml" replacing paths:
```

Comment:
private repo

Reply:
You need to replace it like ```<%= ENV['BUSHSLICER_HOME'] %>/testdata/networking/multus-cni/NetworkAttachmentDefinitions/IPv6/macvlan-bridge-v6.yaml```. Please test it first in your local repo first.

Reply:
@anuragthehatter Update to use <%= ENV['BUSHSLICER_HOME'] %>/testdata/networking/multus-cni/*. Thanks!

Reply:
The testing log with update: https://openshift-qe-jenkins.rhev-ci-vms.eng.rdu2.redhat.com/job/Runner-v3-smoke/856/console